jsonschema2pojo-integration-tests/src/test/java/org/jsonschema2pojo/integration/TypeIT.java

```diff
@@ -282,13 +282,13 @@ public void useDoubleNumbersFalseCausesPrimitiveNumbersToBecomeFloats() throws C
     }
 
     @Test
-    public void unionTypesChooseFirstTypePresent() throws ClassNotFoundException, SecurityException, NoSuchMethodException {
+    public void unionTypesChooseFirstTypePresent() throws ReflectiveOperationException {
```

Comment:
Could we add one more check here, where the `"null"` comes first in the array?

So `["null", "boolean"]` should produce a java.lang.Boolean property.

Reply:
Sure